README.md

```diff
@@ -31,6 +31,236 @@ Application-level tracing for Rust.
 structured, event-based diagnostic information. `tracing` is maintained by the
 Tokio project, but does _not_ require the `tokio` runtime to be used.
 
+## Usage
+
+(The below example is borrowed from the `log` crate's yak-shaving
+[example](https://docs.rs/log/0.4.10/log/index.html#examples), modified to
+idiomatic `tracing`)
+
+In `Cargo.toml`:
+
+```toml
+[dependencies]
+tracing = "0.1"
+tracing-subscriber = "0.2.0-alpha.2"
+```
+
+In `src/main.rs`:
+
+```rust
+mod yak_shave;
+
+use std::{error::Error, io};
+use tracing::{debug, error, info, span, warn, Level};
+use tracing_subscriber::FmtSubscriber;
+use yak_shave;
+
+fn main() {
```

Comment:
maybe the main function _should_ go in the "usage in apps" section? idk.

Reply:
I _did_ opt for moving this for "usage in apps". I figure we can update this again if it's a bit confusing.